docs/install_guide.adoc

```diff
@@ -175,6 +175,86 @@ $ oc describe secret osp-secret -n openstack | grep Watcher
 WatcherPassword:                  9 bytes
 ----
 +
+. Watcher does not support creating certificates for TLS, but assumes that the
+user has created them beforehand. To create the certificates, use the Cert-Manager Operator
+installed by the Openstack Operator and create `Certificate` objects. For TLS
+everywhere, three Certificates are needed, one for the route, one for the
+internal endpoint and one for the public endpoint.
++
+Before creating the certificates, note the project where the openstack
+controlplane is deployed, which by default is __openstack__. When creating the
+certificates, you will need to set the name of the project in the DNS names
+section. Create a `watcher-certificates.yaml` file:
++
+[source,yaml]
+----
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: watcher-internal-svc
+spec:
+  dnsNames:
+  - watcher-internal.openstack.svc
+  - watcher-internal.openstack.svc.cluster.local
+  duration: 43800h0m0s
+  issuerRef:
+    group: cert-manager.io
+    kind: Issuer
+    name: rootca-internal
+  secretName: cert-watcher-internal-svc
+  usages:
+  - key encipherment
+  - digital signature
+  - server auth
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: watcher-public-route
+spec:
+  dnsNames:
+  - watcher-public-openstack.example.com
+  issuerRef:
+    group: cert-manager.io
+    kind: Issuer
+    name: rootca-public
+  secretName: cert-watcher-public-route
+  usages:
+  - key encipherment
+  - digital signature
+  - server auth
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: watcher-public-svc
+spec:
+  dnsNames:
+  - watcher-public.openstack.svc
+  - watcher-public.openstack.svc.cluster.local
+  duration: 43800h0m0s
+  issuerRef:
+    group: cert-manager.io
+    kind: Issuer
+    name: rootca-public
+  secretName: cert-watcher-public-svc
+  usages:
+  - key encipherment
+  - digital signature
+  - server auth
+----
++
+. `oc apply` the file to create the certificates:
++
+[,console]
+----
+$ oc apply -f watcher-certificates.yaml
+certificate.cert-manager.io/watcher-internal-svc created
+certificate.cert-manager.io/watcher-public-route created
+certificate.cert-manager.io/watcher-public-svc created
+----
++
 . Create a file on your workstation named `watcher.yaml` to define the Watcher CR. Although the exact parameters of your
 file may depend on your specific environment customization, a Watcher CR similar to the example below would work in
 a typical deployment:
```

Comment:
Also, note about replace example.com in the endpointURL field.

Reply:
done, thanks!

Reply:
Thanks, I like it!